### ISSUE TITLE
[soundcloud] ERROR: unable to download video data: HTTP Error 403: Forbidden

### ISSUE THREAD

**Issue by pricejn2** [rg3/youtube-dl#5101]
403 Forbidden when trying to download some soundcloud songs. I can browse to the url and listen to the song without any problems.
```
youtube-dl -v https://soundcloud.com/dggtwn/boeboe-subp-yao-clappin
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', 'https://soundcloud.com/dggtwn/boeboe-subp-yao-clappin']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.28
[debug] Python version 3.2.3 - Linux-3.14.32-xxxx-grs-ipv6-64-x86_64-with-Debian-7
[debug] exe versions: avconv 1.2.12, avprobe 1.2.12, ffmpeg 1.2.12, ffprobe 1.2.12, rtmpdump 2.4
[debug] Proxy map: {}
[soundcloud] dggtwn/boeboe-subp-yao-clappin: Resolving id
[soundcloud] dggtwn/boeboe-subp-yao-clappin: Downloading info JSON
[soundcloud] 138349422: Downloading track url
[debug] Invoking downloader on 'https://cf-media.sndcdn.com/cqgfVpoz3oJp.128.mp3?Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiKjovL2NmLW1lZGlhLnNuZGNkbi5jb20vY3FnZlZwb3ozb0pwLjEyOC5tcDMiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjE0MjUyNDgwMDF9fX1dfQ__&Signature=N4GTHMbzc0-Ih1GzgIXx2lGBURKWOxqWWN1ZVJ4QlZlepPUoNahmFeS~W~JR~546j13T735Iqp3Ii4s9iJ-tY9dl6QyaqybCtXkDKcjv00zxV83bniKb1auk2TUMoNWhRyF~nqf3dWdu2rnW5tkH-FSnCmHk3LYHzUOYrQZNEti8z9b-xsllAk9D2G6GIsIqExOHnKv4QjUgUzSS0B2-xu~-nuzIaQ75BddQA6n5KHdqG9WoQmmjVeYHx8~5imMRee8Ay99uCnQE~~uWVWKkOaWk9j807epgRx8WC2LOx49BzihcZZxPeYOay6txK5Vll57qDDG~C34NWiNyv458Sw__&Key-Pair-Id=APKAJAGZ7VMH2PFPW6UQ'
ERROR: unable to download video data: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "/usr/local/lib/python3.2/dist-packages/youtube_dl/YoutubeDL.py", line 1375, in process_info
    success = dl(filename, info_dict)
  File "/usr/local/lib/python3.2/dist-packages/youtube_dl/YoutubeDL.py", line 1350, in dl
    return fd.download(name, info)
  File "/usr/local/lib/python3.2/dist-packages/youtube_dl/downloader/common.py", line 339, in download
    return self.real_download(filename, info_dict)
  File "/usr/local/lib/python3.2/dist-packages/youtube_dl/downloader/http.py", line 64, in real_download
    data = self.ydl.urlopen(request)
  File "/usr/local/lib/python3.2/dist-packages/youtube_dl/YoutubeDL.py", line 1666, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python3.2/urllib/request.py", line 376, in open
    response = meth(req, response)
  File "/usr/lib/python3.2/urllib/request.py", line 488, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.2/urllib/request.py", line 414, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.2/urllib/request.py", line 348, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.2/urllib/request.py", line 496, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

Thanks!

**Comment by dstftw** [comment]
This will be fixed in the next version. Thanks for the report.

**Comment by pricejn2** [comment]
Awesome! Thanks @dstftw !

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.03.03 and newer. See [our FAQ](https://yt-dl.org/update) if you need help updating.
